vars/createGithubIssue.groovy

```diff
@@ -22,7 +24,7 @@ void call(Map args = [:]) {
     try {
         withCredentials([usernamePassword(credentialsId: 'jenkins-github-bot-token', passwordVariable: 'GITHUB_TOKEN', usernameVariable: 'GITHUB_USER')]) {
             def openIssue = sh(
-                    script: "gh issue list --repo ${args.repoUrl} -S \"${args.issueTitle} in:title\" --label ${label} --json number --jq '.[0].number'",
+                    script: "gh issue list --repo ${args.repoUrl} -S \"${args.issueTitle} in:title\" --label \"${label}\" --json number --jq '.[0].number'",
```

Comment:
In case there are multiple issues returned then is this picking up the top result?

Reply:
Yes Risab that was the idea for this existing library `createGithubIssue`, the logic was with same title for all the AUTOCUT issues, there will be only one issue created for one scenario.